client/blocks/jitm/templates/modal-style.scss

```diff
@@ -121,30 +121,35 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 	@media ( min-width: $wpcom-modal-breakpoint ) {
 		flex-direction: column;
 		justify-content: flex-end;
-		max-width: 60%;
 		min-height: $wpcom-modal-content-min-height;
 		bottom: 0;
+		padding: 0 2% 0 7%;
 	}
 }
 
 .modal__text,
 .modal__visual {
 	@media ( min-width: $wpcom-modal-breakpoint ) {
 		flex: 1 auto;
-		min-width: 290px;
 	}
 }
 
 .modal__text {
-	padding: 1em 2em 1em 0;
+	padding: 1em 0;
```

Comment:
Rely on `.modal__container` to set consistent left and right padding for both the text and heading.